libs/domain/site/dropdown-item/dropdown-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ContentMixin } from '@spryker-oryx/experience';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { LitElement, TemplateResult, html } from 'lit';
+import { when } from 'lit/directives/when.js';
+import {
+  DropdownItemContent,
+  DropdownItemOptions,
+} from './dropdown-item.model';
+import { styles } from './dropdown-item.styles';
+
+export class DropdownItemComponent extends ContentMixin<
```

Comment:
Also I'm wondering why it's in the site domain and can we make it a general component inside ui?

Reply:
UI components are dummy, but this one uses content mixin under the hood (with content service) and can be used in EB compositions. This is definitely domain component

Reply:
It looks like a UI component to me, but details of the implementation should be clarified by @tobi-or-not-tobi together with Helen.